src/Components/Results.js

```diff
@@ -1,12 +1,20 @@
+/* eslint-disable react/jsx-key */
 /* eslint react/prop-types: 0 */
 import React from 'react';
+import { BrowserRouter, Link } from 'react-router-dom';
 
 const Results = (props) => {
   const { repos } = props;
 
   const listRepos =
     repos.length !== 0 ? (
-      repos.data.map((item) => <li key={item.id}> {item.name} </li>)
+      repos.data.map((item) => (
+        <BrowserRouter>
+          <Link to={{ pathname: `/repo/${item.full_name}` }} key={item.id}>
+            <li>{item.name} </li>
+          </Link>
+        </BrowserRouter>
```

Comment:
You do not need BrowserRouter here. It is just a simple Link.
```
 repos.data.map((item) => (
        <Link to={{ pathname: `/repo/${item.full_name}` }} key={item.id}>
          <li>{item.name} </li>
        </Link>
      ))
```

Reply:
When I delete it I get an error

![image](https://user-images.githubusercontent.com/47263506/108244315-607b9900-714f-11eb-9bde-cf2d5381db3e.png)